.gitignore

```diff
@@ -1,2 +1,5 @@
 vendor
 .DS_Store
+.vscode
```

Comment:
you dont want to store vscode config? This can be beneficial in creating a consistent devEx

Reply:
Oh, we could keep it. I deleted it because it just had some spell check options set, seemed like it might have been an accidental include